src/js/plugins/plugin.vendor-fullscreen.js

```diff
@@ -0,0 +1,231 @@
+/* global BookReader */
+
+/**
+ * Toggles browser's native fullscreen mode if available
+ */
+
+const EVENT_NAMESPACE = '.bookreader_vendor-fullscreen';
+// function isMobile() {
+//   return (typeof window.orientation !== 'undefined') || (navigator.userAgent.indexOf('IEMobile') !== -1);
+// };
+
+// if (this.isMobile()) { return; }
+
+jQuery.extend(BookReader.defaultOptions, {
+  /** @type {true} */
```

Comment:
Only use `type {true}` if it _can only_ be `true`; this can be `false` it's just been initialized to `true`

```suggestion
  /** @type {boolean} */
```